bin/templates/cordova/handler.js

```diff
@@ -121,9 +121,12 @@ module.exports = {
         install: (asset, plugin_dir, wwwDest) => {
             const src = path.join(plugin_dir, asset.src);
             const dest = path.join(wwwDest, asset.target);
+            const destDir = path.parse(dest).dir;
+
+            if (destDir) fs.ensureDirSync(destDir);
 
             if (fs.statSync(src).isDirectory()) {
-                fs.copySync(src + '/*', dest);
+                fs.copySync(src, dest);
```

Comment:
Does this still need to be told to copy all the directory contents, or does that happen automatically? If it happens automatically, is there any need for the `if`/`else` here since they seem to be running the same code?

Reply:
From `fs-extra` docs.
```
src <String> Note that if src is a directory it will copy everything inside of this directory, not the entire directory itself (see issue #537).
```

Before, the `fs.copySync`  was a `shell.cp` that needed that pattern.

Reply:
We might be able to remove the `if` statement that you pointed out. `copySync` will handle both file and directory. The statement I believe was just original code that can be cleanup.

Reply:
Please see the latest update for the refactor. I dropped the if statement and re-ran the mobilespec to ensure it continues to build successfully.